packages/devtools_app/build.yaml

```diff
@@ -1,7 +0,0 @@
-targets:
```

Comment:
Other thing to call out is we are losing the -O1 flag for dart2js for our deployed app which is fine. We were a bit of an outlier deploying an app with type checks in Dart2JS. Likely the timeline page and the inspector are slightly faster with this flag but we are missing performance benchmarks to verify.

Reply:
I've restored this file, so we should get the same flags back.

Reply:
You can build a "profile" release build for web which is O1

Reply:
Interesting. What is "release"? If Release was O3 but "profile" was O3 that would lead to potential significant divergence in performance on the web.  O1 includes costly type checks O3 omits.

Reply:
04. TBD how we map O(N) to profile/release